packs/spell-effects/spell-effect-hymn-of-healing.json

```
@@ -4,7 +4,7 @@
     "name": "Spell Effect: Hymn of Healing",
     "system": {
         "description": {
-            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Item.Hymn of Healing]</em></p>\n<p>Your divine singing mends wounds and provides a temporary respite from harm. The target gains fast healing 2. When you Cast the Spell and the first time each round you Sustain the Spell, the target gains 2 temporary Hit Points, which last for 1 round.</p>"
+            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Item.Hymn of Healing]</em></p>\n<p>Your glorious singing mends wounds and provides a temporary respite from harm. The target gains fast healing 2. When you Cast the Spell and the first time each round you Sustain the Spell, the target gains 2 temporary Hit Points, which last for 1 round.</p>"
```

```suggestion
            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Item.Hymn of Healing]</em></p>\n<p>The target gains fast healing 2. When you Cast the Spell and the first time each round you Sustain the Spell, the target gains 2 temporary Hit Points, which last for 1 round.</p>"
```
In general we don't need the flavor text in the effect anyways. Also we're doing the temp HP wrong on this, but there's not a great way to do them.